pkg/netutils/netutils.go

```diff
@@ -107,10 +109,11 @@ func NetLimit() string {
 	}
 
 	if maxInterfaceLimit > 0 {
-		return strconv.FormatUint(maxInterfaceLimit/8, 10) + "M"
+		r := model.Rate(maxInterfaceLimit)
```

Comment:
If the maxInterfaceLimit speed is `1000Mb/s`, and then you will get `800B` here.

Reply:
I will fix this tomorrow